vision/classification/efficientnet-lite4/README.md

```diff
@@ -0,0 +1,108 @@
+# EfficientNet-Lite4
+
+## Use-Cases
+EfficientNet-Lite4 is an image classification model that achieves state-of-the-art accuracy. It is designed to run on mobile CPU, GPU, and EdgeTPU devices, allowing for applications on mobile and loT, where computational resources are limited.
+
+## Description
+EfficientNet-Lite4 is a version of EfficentNet-Lite, where it is the largest variant, integer-only quantized model that produces the highest accuracy. ImageNet top-1 accuracy, while still running in real-time (e.g. 30ms/image) on a Pixel 4 CPU. 
```

Comment:
nit: EfficientNet-Lite 4 is the largest variant and most accurate of the set of EfficientNet-Lite model. It is an integer-only quantized model that produces the highest accuracy of all of the EfficientNet models.

Second sentence also needs to be reworded -- I think it's missing some context.